### PR TITLE
Sort websites with shared credential backends

### PR DESCRIPTION
**Pull request by darinadler**
### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically 
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

### CODE REVIEW

File: quirks/websites-with-shared-credential-backends.json [hunk 1]
```diff
@@ -1,273 +1,274 @@
...
+        "kp.org"
     ],
     [
-        "alltrails.com",
-        "alltrails.io"
+        "live.com",
+        "microsoft.com",
+        "microsoftonline.com",
+        "office.com",
+        "skype.com"
```
Comment (darinadler) [inline quirks/websites-with-shared-credential-backends.json:206]: This used to be two separate lists, both including live.com. This merges them.

**Comment by igor-makarov** [comment]
Thanks for the initiative!
However, #130 covers the sorting by using a script. #151 covers the duplicate checks. 

Consider closing the PR in favor of those?

**Comment by darinadler** [comment]
I’ll certainly drop this if those get merged!

**Comment by igor-makarov** [comment]
Yeah, they got held up because GH Actions aren't enabled for this repo yet...

**Pull request closed without merging**